src/rgw/setup/rgw.py

```diff
@@ -452,7 +441,7 @@ def _create_rgw_user(conf: MappedConf):
                     retry_count = retry_count + 1
                     continue
                 else:
-                    Log.error(f'"{create_usr_cmd}" failed with error {err}.')
+                    Log.error(f'RGW admin user creation failed with error {err}.')
```

Comment:
good you handled here too. Someone looking at the code can figure this out, but he/she will have to feed some values to it, which would take some efforts still. For debugging, we can do this feeding and figure out still